src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2807,13 +2807,25 @@ class S4Importer final : public IParkImporter
                             ConvertWall(&type, &colourA, &colourB);
 
                             type = _wallTypeToEntryMap[type];
-                            auto edgeSlope = LandSlopeToWallSlope[slope][edge & 3] & ~EDGE_SLOPE_ELEVATED;
+                            auto baseZ = originalTileElement.GetBaseZ();
+                            auto clearanceZ = originalTileElement.GetClearanceZ();
+                            auto edgeSlope = LandSlopeToWallSlope[slope][edge & 3];
+                            if (edgeSlope & EDGE_SLOPE_UPWARDS || edgeSlope & EDGE_SLOPE_DOWNWARDS)
```

Comment:
```suggestion
                            if (edgeSlope & (EDGE_SLOPE_UPWARDS || EDGE_SLOPE_DOWNWARDS))
```
Just a suggestion

Reply:
That's not going to work like that - you need a bitwise or, not a boolean or.

Reply:
```suggestion
                            if (edgeSlope & (EDGE_SLOPE_UPWARDS | EDGE_SLOPE_DOWNWARDS))
```